### PR TITLE
Documentation improvements for 0.3

### PR DESCRIPTION
**Pull request by ageis**

### CODE REVIEW

File: docs/install.md [hunk 1]
```diff
@@ -358,52 +358,96 @@ Once the installation is complete, the addresses for each Tor Hidden Service wil
...
+
+### Set up SSH host aliases
+
+This step is optional but makes it much easier to connect to and administer the servers. Create the file `/home/amnesia/.ssh/config` and add a configuration following the scheme below, but replace `Hostname` and `User` with the values specific to your setup:
+
+```
+Host app
+  Hostname fsrrszf5qw7z2kjh.onion
+  User <ssh_user>
+  ProxyCommand connect -R remote -5 -S localhost:9050 %h %p
```
Comment (garrettr) [inline docs/install.md]: So is connect-proxy available on Tails by default?

Comment (ageis) [inline docs/install.md]: Yes, it is.

**Comment by garrettr** [comment]
This looks great! Just one request: can you remove all of the trailing whitespace that this adds?

**Comment by ageis** [comment]
What do you mean? I'm not seeing it, please show me.

**Comment by garrettr** [comment]
This PR actually only adds a handful of instances, although the diff with 0.3 has quite a bit. Here's a few examples: [1](https://github.com/freedomofpress/securedrop/pull/927/files#diff-64407d3e6d5ef382426bdfbcae040c6aR59), [2](https://github.com/freedomofpress/securedrop/pull/927/files#diff-c474cab07b2dd9f949f47376f7d47fd2R79), [3](https://github.com/freedomofpress/securedrop/pull/927/files#diff-c474cab07b2dd9f949f47376f7d47fd2R109).

**Pull request merged**